gravitee-apim-console-webui/src/management/application/details/user-group-access/members/application-general-members.component.ts

```diff
@@ -251,6 +251,10 @@ export class ApplicationGeneralMembersComponent {
     }
   }
 
+  public isInProgressMember(memberId: string) {
```

Comment:
I don't quite understand "in progress member" 🤔 what this mean ?

Reply:
Okay, so there's an issue when adding new member that wasn't saved yet I could press 'delete' button next to it and it was deleting a member that doesn't exist yet. This method looks for the member in membersList to make sure it exists. Maybe the naming is not the best 😅

<img width="946" alt="Screenshot 2024-01-16 at 5 22 53 PM" src="https://github.com/gravitee-io/gravitee-api-management/assets/144100648/e09d3c24-5195-4dbb-938e-a8d3ae73aa03">
On that screenshot last member is not saved yet so I can't delete it

Reply:
Ok ok 
what do you think about adding a optional flag in MemberDataSource like `notSaved: boolean`
To avoid this function ? 

but i find it strange that i can't delete the member i've just added. perhaps this should be handled in the remove function without a confirmation dialog.

p.s : could you remove all `: any` type please we use typescript and not javascript, so in 99% of cases there is no reason to use it (found one here : `public removeMember(member: any) {`)

Reply:
Changes added

Reply:
not totally. 
We need to be able to continue deleting the added member recently